lib/react/rails/test_helper.rb

```diff
@@ -0,0 +1,23 @@
+module React
+  module Rails
+    module TestHelper
+      extend ActiveSupport::Concern
+
+      # assert react_component render
+      #
+      # assert_react_component("HelloWorld") do |props|
+      #   assert_equal "Hello world", props[:message]
+      # end
+      def assert_react_component(name)
```

Comment:
Would it be useful and/or possible to also allow assertion if it is prerendered or not?

Reply:
This my first use react-rails and I am not use `prerender` feature, so I don't know the render logic, would you please add `prerender` support after the PR merged?